examples/reshare/src/dkg/actor.rs

```diff
@@ -186,15 +193,20 @@ where
                 (0, initial_public, initial_share)
             };
         let all_participants = Self::collect_all(&active_participants, &inactive_participants);
-        let (active_participants, inactive_participants) = Self::select_participants(
+        let (dealers, mut players) = Self::select_participants(
             current_epoch,
             self.num_participants_per_epoch,
             active_participants,
             inactive_participants,
         );
 
+        // If we're performing DKG, dealers == players.
+        if self.is_dkg {
```

Comment:
Can we just infer this if `initial_public` is `None`?

Reply:
This is what we already did, but we can do it outside of the config.